kite-data/kite-data-crunch/src/main/java/org/kitesdk/data/crunch/DatasetTarget.java

```diff
@@ -85,7 +86,12 @@ public boolean handleExisting(WriteMode writeMode, long lastModForSource,
       }
     }
 
-    boolean exists = !view.isEmpty();
+    boolean ready = false;
+    if(view instanceof ReadiableView) {
```

Comment:
This should cover all of them, 7392007. Hadn't noticed I was dropping the spaces.